src/tools/fsync/test_fsync.c

```diff
@@ -69,7 +69,7 @@ main(int argc, char *argv[])
 	for (i = 0; i < XLOG_SEG_SIZE; i++)
 		full_buf[i] = 'a';
 
-	if ((tmpfile = open(filename, O_RDWR | O_CREAT, S_IRUSR | S_IWUSR, 0)) == -1)
```

Comment:
Further, I tested using Chef for CentOS and Ubuntu; but verifying man pages I see the following for [Mac OS X](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man2/open.2.html) (which does not offer a free OSS test bed that I know of) versus [Ubuntu](http://manpages.ubuntu.com/manpages/lucid/man2/open.2.html) which looks to be from upstream Linux.